nextflow.config

```diff
@@ -252,45 +259,153 @@ docker.registry      = 'quay.io'
 podman.registry      = 'quay.io'
 singularity.registry = 'quay.io'
 
-def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_timeline_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_timeline_${params.trace_report_suffix}.html"
 }
 report {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_report_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/execution_report_${params.trace_report_suffix}.html"
 }
 trace {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/execution_trace_${trace_timestamp}.txt"
+    file    = "${params.outdir}/pipeline_info/execution_trace_${params.trace_report_suffix}.txt"
 }
 dag {
     enabled = true
-    file    = "${params.outdir}/pipeline_info/pipeline_dag_${trace_timestamp}.html"
+    file    = "${params.outdir}/pipeline_info/pipeline_dag_${params.trace_report_suffix}.html"
 }
 
 manifest {
     name            = 'nf-core/crisprseq'
-    author          = """Júlia Mir Pedrol, Laurence Kuhlburger"""
+    author          = """Júlia Mir Pedrol, Laurence Kuhlburger""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
+    contributors    = [
+        // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
+        [
+            name: 'Júlia Mir Pedrol',
+            affiliation: 'Centre for Genomic Regulation (CRG)',
+            email: 'mirp.julia@gmail.com',
+            github: 'https://github.com/mirpedrol',
+            contribution: ['author', 'maintainer'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: 'https://orcid.org/0000-0001-6104-9260'
+        ],
+        [
+            name: ' Laurence Kuhlburger',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/LaurenceKuhl',
+            contribution: ['author', 'maintainer'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Matthias Hörtenhuber',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/mashehu',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
+        [
+            name: 'Alan Tracey',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/alan-tracey',
+            contribution: ['contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
+        ],
```

Comment:
Hi @alan-tracey,
we have added all the contributors to the pipeline and you contributed to crisprseq. Could you please update your data here? 
Thanks!